sixtyfps_compiler/object_tree.rs

```diff
@@ -396,7 +396,7 @@ impl<'a> IntoIterator for &'a BindingsMap {
     type IntoIter = std::collections::btree_map::Iter<'a, String, BindingExpression>;
 
     fn into_iter(self) -> Self::IntoIter {
-        (&self.0).into_iter()
+        (&self.0).iter()
```

Comment:
not sure what was wrong with that?
then maybe we don't need to take a reference anymore?

Reply:
This is why this is bad (according to the clippy help):

> Readability. Calling into_iter on a reference will not move out its content into the resulting iterator, which is confusing. It is better just call iter or iter_mut directly.